i18n/en.json

```diff
@@ -8,5 +8,7 @@
 	"ext-whowrotethat-state-error": "Error: $1",
 	"ext-whowrotethat-state-error-generic": "Could not load WhoWroteThat. Please refresh and try again.",
 	"ext-whowrotethat-ready-title": "Who Wrote That?",
-	"ext-whowrotethat-ready-general": "Hover to see contributions by the same author. Click for more details."
+	"ext-whowrotethat-ready-general": "Hover to see contributions by the same author. Click for more details.",
+
+	"ext-whowrotethat-requested-data-is-not-currently-available-in-wikiwho-database-it-will-be-available-soon": "WikiWho data is not currently available for this article. Please try again soon."
```

Comment:
That's....... a tad long for a key 😝 

Let's make it something more standardized, like `ext-whowrotethat-api-error-wikiwho-generic` ? or something like that?